core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpExecConfig.scala

```diff
@@ -0,0 +1,38 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatterplot
+
+import edu.uci.ics.amber.engine.architecture.breakpoint.globalbreakpoint.GlobalBreakpoint
+import edu.uci.ics.amber.engine.architecture.deploysemantics.deploymentfilter.{UseAll}
+import edu.uci.ics.amber.engine.architecture.deploysemantics.deploystrategy.{RoundRobinDeployment}
+import edu.uci.ics.amber.engine.architecture.deploysemantics.layer.WorkerLayer
+import edu.uci.ics.amber.engine.common.virtualidentity.util.makeLayer
+import edu.uci.ics.amber.engine.common.virtualidentity.{ActorVirtualIdentity, OperatorIdentity}
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.tuple.schema.OperatorSchemaInfo
+
+class ScatterplotOpExecConfig(
+    tag: OperatorIdentity,
+    opDesc: ScatterplotOpDesc,
+    numWorkers: Int,
+    schemaInfo: OperatorSchemaInfo
+) extends OpExecConfig(tag) {
+
+  override lazy val topology = new Topology(
+    Array(
+      new WorkerLayer(
+        makeLayer(tag, "main"),
+        _ => new ScatterplotOpExec(opDesc, schemaInfo),
+        numWorkers,
+        UseAll(),
```

Comment:
I remember you mentioning this will always use 1 worker, so why did you change this to `UseAll`? Does this mean is will use more workers now?

Reply:
For spatial point visualization, we will use a single worker(pixel based). For normal scatterplot, points will be projected as is(no pixel based logic) so number of workers can be many

Reply:
👌